PeriodicMeetings/2023-07-10.md

```diff
@@ -0,0 +1,76 @@
+# Periodic Meeting Notes
+
+## What I accomplished last week
+
+[Link to last meeting notes](./2023-07-03.md)<!-- Edit this link to the date of the last meeting -->
+
+### Priority
+
+#### A
+
+- [X] Port Clustal-Omega to Yggdrasil 
+    - [X] Port libargtable2 to Yggdrasil (NEW PR - https://github.com/JuliaPackaging/Yggdrasil/pull/7021)
+    - [X] Port/merge ClustalO to Yggdrasil (https://github.com/JuliaPackaging/Yggdrasil/pull/7023)
+- [X] Initial ClustalO.jl repo (https://github.com/M-PERSIC/ClustalO.jl)
+- [X] Initial MAFFT.jl repo (https://github.com/M-PERSIC/MAFFT.jl)
+
+#### B
+
+- [X] Launch BioJuliaDocs community initiative (https://github.com/M-PERSIC/BioJuliaDocs.git)
+    - [X] Setup repo (pull request approvals, actions,...)
+    - [X] Generate GitHub project board
+    - [X] Invite community (Slack, Discourse)
+
+#### Other
+
+- [X] First pull request to BioJulia
+    - [X] Enable `documenter-dark` theme for ReadDatastores.jl (https://github.com/BioJulia/ReadDatastores.jl/pull/19)
+- [X] Contact @maltesie for potential [BioTutorials](https://github.com/BioJulia/BioTutorials) integration (no word yet)
+- [X] Contact @kool7d for potential [BioMakie](https://github.com/kool7d/BioMakie.jl) integration
+- [ ] Generate Bowtie/Bowtie2 JLL package 
+    - [X] Add libsais dependency (https://github.com/JuliaPackaging/Yggdrasil/pull/7029)
+    - [X] Add SIMDe dependency (awaiting merge - https://github.com/JuliaPackaging/Yggdrasil/pull/7035)
+    - [ ] Generate Bowtie2 build recipe
+    - [ ] Generate Bowtie build reicpe
+
+### Notes
+
+#### A
+
+- I'm pushing back the previous MUSCLE.jl goals to next week. I want to dedicate more time to studying BioPython/BioConductor implementations of MSA as well as good patterns for wrappers of external programs 
```

Comment:
That seems fine, but don't look too hard at other implementations, as they won't be very Julian